src/mlj_flux.jl

```diff
@@ -219,22 +219,6 @@ function MLJFlux.train(
 )
     X = X isa Tables.MatrixTable ? MLJBase.matrix(X) : X
 
-    if !isa(chain, AbstractLaplace)
-        la = LaplaceRedux.Laplace(
-            chain;
-            likelihood=:regression,
-            subset_of_weights=model.subset_of_weights,
-            subnetwork_indices=model.subnetwork_indices,
-            hessian_structure=model.hessian_structure,
-            backend=model.backend,
-            σ=model.σ,
-            μ₀=model.μ₀,
-            P₀=model.P₀,
-        )
-    else
-        la = chain
-    end
-
     # Initialize history:
```

Comment:
@MojiFarmanbar  honestly i do not think it would work because 
MLJFlux.train_epoch(
            model, chain, regularized_optimiser, optimiser_state, X, y
        ) requires an actual chain not a laplace object

the laplace object is required only by LaplaceRedux.fit which is at the end.

have you tested it?